pset-iterators.js

```diff
@@ -92,6 +106,12 @@ Tip: your new array should contain 45 elements
 console.log('Problem 3:')
 
 // Add your code below this line
+const presidentialParties = presidents.map(president => {
+  return president.party
+})
+
+console.log(presidentialParties)
+//console.log(presidentialParties.length)
```

Comment:
@nmikhol Good job here using the map method to return the president parties in an array.

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L94)